matrix_sdk/examples/command_bot.rs

```diff
@@ -1,3 +1,7 @@
+// allow biggr types until rustc fix, consider removing after new rust
+// stable release. see https://github.com/rust-lang/rust/issues/54540
+#![type_length_limit = "1075569"]
```

Comment:
This seems to be unneeded now that Rust 1.47.0 got released. :sweat_smile:

Reply:
meh, I just downloaded the latest stable again with rustup a couple of hours ago as I was using distro stuff before trying nightly, and it wasn't available yet!! heh.

Will the travis environment have 1.47 right away?

Reply:
Indeed it will, it fixed the CI on the crypto-improvements branch as well.